dev/windows/setup.ps1

```diff
@@ -0,0 +1,36 @@
+$phpDir = Read-Host "Enter the path of your PHP installation (e.g. 'C:\Program Files\php', 'D:\php', 'C:\wamp\bin\php')"
+$fullPhpDir = Join-Path $phpDir "php.exe"
+
+# check if path exists
+if (-Not (Test-Path $phpDir)) {
+  throw "The directory '$phpDir' does not exist. Did you point to the exact file instead of the containing folder?"
+}
+
+# check if php.exe is there
+if (-Not (Test-Path $fullPhpDir)) {
+  Write-Error "'php.exe' was not found in '$phpDir'. Was it installed correctly?"
+  exit 1
```

Comment:
In the guard clause above this, you `throw` instead of a `Write-Error` followed by `exit 1`. `throw` is appropriate in this guard clause as well.

```suggestion
  throw "'php.exe' was not found in '$phpDir'. Was it installed correctly?"
```